### PR TITLE
Add recipe: ncbi-vdb-py

### PR DESCRIPTION
**Pull request by luizirber**
Add a new recipe for the Python bindings of `ncbi-vdb`.

~~This is a weird recipe, because the bindings are in `./py_vdb/vdb.py` in the original source for `ncbi-vdb`, and since it is ctypes-based there is no package or external dependencies, it is a single-file module. So I checked the `snakemake` recipe and added another output to the `ncbi-vdb` recipe.~~
I ended up creating a separate recipe, it was getting too complicated to maintain `outputs`.

(I also copied the `vdb.py` directly into the `site-packages` for the installed Python, which feels incredibly weird but has precedents in other recipes in bioconda)

Does this look correct?

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by luizirber** [comment]
@BiocondaBot please fetch artifacts

**Comment by luizirber** [comment]
@BiocondaBot please fetch artifacts

**Comment by luizirber** [comment]
@bioconda/core calling for help on this one. I think I mirrored the `snakemake` recipe to set up `ncbi-vdb` and `ncbi-vdb-py`, but seems only the latter is generating packages?

**Comment by luizirber** [comment]
@BiocondaBot please fetch artifacts

File: recipes/ncbi-vdb/meta.yaml [hunk 1]
```diff
@@ -41,3 +42,23 @@ test:
...
+    requirements:
+      run:
+        - ncbi-vdb ={{ version }}
+        - python >=3.5
+
+    test:
+      imports:
+        - vdb
+      commands:
+        - python -c "import vdb; vdb.manager()"
```
Comment (luizirber) [inline recipes/ncbi-vdb/meta.yaml:64]: I tried this to test if the module is working, but `python` here is the system python, not the conda python. What is the proper way for calling the right Python?

Comment (luizirber) [reply recipes/ncbi-vdb/meta.yaml:64]: (see failed run in https://app.circleci.com/pipelines/github/bioconda/bioconda-recipes/33633/workflows/6328262e-54c5-48e5-a3b9-5d1c70e716f0/jobs/125609)

**Comment by luizirber** [comment]
@BiocondaBot please fetch artifacts

**Comment by luizirber** [comment]
@BiocondaBot please fetch artifacts

**Review by ctb — APPROVED** [review]

**Comment by luizirber** [comment]
@BiocondaBot please merge

**Pull request merged**